proto/provenance/name/v1/name.proto

```diff
@@ -36,26 +36,6 @@ message NameRecord {
   bool restricted = 3;
 }
 
-// CreateRootNameProposal details a proposal to create a new root name
-// that is controlled by a given owner and optionally restricted to the owner
-// for the sole creation of sub names.
-message CreateRootNameProposal {
-  option (gogoproto.equal)            = false;
-  option (gogoproto.goproto_getters)  = false;
-  option (gogoproto.goproto_stringer) = false;
-
-  // proposal title
-  string title = 1;
-  // proposal description
-  string description = 2;
-  // the bound name
-  string name = 3;
-  // the address the name will resolve to
-  string owner = 4;
-  // a flag that indicates if an owner signature is required to add sub-names
-  bool restricted = 5;
-}
-
```

Comment:
I'm not sure we want to remove these from the proto file.  Did any of these proposals get executed in `mainnet`?

My thought here is, we want to keep things around for the proto library that gets published.  So when third party apps, i.e. the explorer pulls in the never proto version it is still able to deserialize old proto objects.  The data projects will also benefit from this.

Reply:
Thanks for finding this, added back.